Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcServer : IRpcServer
+    {
+        private readonly INetqRpcRabbitConnection<Func<Type,object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        private IDisposable _respondHandler;
```

Comment:
```suggestion
        private readonly INetqRpcRabbitConnection<Func<Type,object>> _netqRpcRabbitConnection;
        private IDisposable _respondHandler;
        private readonly ILogger _logger;
```

I looks more good like that and have some sort of logic, they ordered like the order you use them. And i think that the logger should always be at the end